backend/api/auth_endpoints.py

```
@@ -0,0 +1,281 @@
+            )
+
+    except Exception as e:
+        logger.error(f"Error during login: {e}")
+        raise HTTPException(status_code=500, detail="Authentication service error")
+
+
+@auth_router.post("/social-login", response_model=AuthResponse)
+async def social_login(request: SocialLoginRequest) -> AuthResponse:
+    """
+    Handle social login (OAuth) from providers like Google, Discord, etc.
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)